crates/configuration/src/parachain.rs

```diff
@@ -20,118 +20,108 @@ pub enum RegistrationStrategy {
     is_cumulus_based: bool,
-
-    /// List of parachain's bootnodes addresses to use.
     bootnodes_addresses: Vec<Multiaddr>,
-
-    /// List of parachain's collators to use.
     collators: Vec<NodeConfig>,
 }
 
 impl ParachainConfig {
+    /// The parachain ID.
     pub fn id(&self) -> u32 {
         self.id
     }
 
+    /// The chain name.
     pub fn chain(&self) -> Option<&Chain> {
         self.chain.as_ref()
     }
 
+    /// The registration strategy for the parachain.
     pub fn registration_strategy(&self) -> Option<&RegistrationStrategy> {
         self.registration_strategy.as_ref()
     }
 
+    /// The initial balance of the parachain account.
     pub fn initial_balance(&self) -> u128 {
         self.initial_balance
     }
 
+    /// The default command used for collators.
     pub fn default_command(&self) -> Option<&Command> {
         self.default_command.as_ref()
     }
 
+    /// The default container image used for collators.
     pub fn default_image(&self) -> Option<&Image> {
         self.default_image.as_ref()
     }
 
+    /// The default resources limits used for collators.
     pub fn default_resources(&self) -> Option<&Resources> {
         self.default_resources.as_ref()
     }
 
+    /// The default database snapshot location that will be used for state.
     pub fn default_db_snapshot(&self) -> Option<&AssetLocation> {
         self.default_db_snapshot.as_ref()
     }
 
+    /// The default arguments that will be used to execute the collator command.
     pub fn default_args(&self) -> Vec<&Arg> {
         self.default_args.iter().collect::<Vec<&Arg>>()
     }
 
+    /// The location of a pre-existing genesis WASM runtime blob of the parachain.
     pub fn genesis_wasm_path(&self) -> Option<&AssetLocation> {
         self.genesis_wasm_path.as_ref()
     }
 
+    /// The generator command used to create the genesis WASM runtime blob of the parachain.
     pub fn genesis_wasm_generator(&self) -> Option<&Command> {
         self.genesis_wasm_generator.as_ref()
     }
 
+    /// The location of a pre-existing genesis state of the parachain.
     pub fn genesis_state_path(&self) -> Option<&AssetLocation> {
         self.genesis_state_path.as_ref()
     }
 
+    /// The generator command used to create the genesis state of the parachain.
     pub fn genesis_state_generator(&self) -> Option<&Command> {
         self.genesis_state_generator.as_ref()
     }
 
+    /// The location of a pre-existing chain specification for the parachain.
     pub fn chain_spec_path(&self) -> Option<&AssetLocation> {
         self.chain_spec_path.as_ref()
     }
 
+    /// Whether the parachain is based on cumulus.
     pub fn is_cumulus_based(&self) -> bool {
         self.is_cumulus_based
     }
 
+    /// The bootnodes addresses the collators will connect to.
+    // TODO: is it some kind of default that should be merged with the override in collators ?
```

Comment:
wdyt ?

Reply:
No, we don't have a default here. If any bootnode is provider we will include the first collator as bootnode for the rest of the collators.